### DISCUSSION TITLE
How to terminate / fail cypress if any of the test case takes more than 10 seconds.

### DISCUSSION THREAD

**Discussion by aqilusman45** [cypress-io/cypress#17188] [Questions and Help] [answered]
I looking to setup a cypress test for our app that monitors app performance. For this we will by running e2e that runs after certain interval of time and checks if the app is working fine and if the interaction with the app has not slowed down. For this we need to terminate / fail the cypress test if any of the test case is taking more than 10 seconds. Also I wanted to know how can I trigger a webhook that sends the result of the test results to a slack channel.

Any help on this will be appreciated. Thanks!

**Comment by bahmutov** [comment, marked as answer]
See recipe https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/fundamentals__timeout

**Reply by aqilusman45** [reply]
Hi @bahmutov, thanks for the valuable feedback, this is exactly what I was looking for.
